htdp-lib/2htdp/private/check-aux.rkt

```diff
@@ -8,7 +8,14 @@
 
 (provide (all-defined-out))
 
-(define INSET  5)     ;; the space around the image in the canvas
+(define universe-inset  ;; the space around the image in the canvas
+  (make-parameter
+   5
+   (λ (v)
+     (unless (and (integer? v)
+                  (<= 0 v 1000))
```

Comment:
`1000` really?   Did you try to run a `big-bang` program with such an inset? 

(Very minor: this is a `world-inset` not a `universe-insert`.)